File: C# Advanced/C# Advanced - Exercises/Exercise-05-Functional Programming/Task05.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Practice
{
    class Program
    {
        static void Main(string[] args)
        {
            List<int> numbers = Console.ReadLine().Split().Select(int.Parse).ToList();

            string input = string.Empty;
            while ((input = Console.ReadLine()) != "end")
            {
                if (input == "add")
                {
                    numbers = numbers.Select(x => x + 1).ToList();
                }
                else if (input == "multiply")
                {
                    numbers = numbers.Select(x => x * 2).ToList();
                }
                else if (input == "subtract")
                {
                    numbers = numbers.Select(x => x - 1).ToList();
                }
                else if (input == "print")
                {
                    Console.WriteLine(String.Join(" ", numbers));
                }
            }
        }
    }
}

File: JS Fundamentals/JS Fundamentals - Labs/Lab-05-Arrays Advanced/Task05.js
function solve(numbers) {
    console.log(
        numbers
            .filter(n => n % 2 != 0)
            .map(n => n * 2)
            .sort((a, b) => a - b)
            .join(' ')
    );
}

solve([3, 0, 10, 4, 7, 3]);
